subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/PlayServices.kt

```diff
@@ -1,30 +1,14 @@
 package com.avito.android.runner
 
 import android.content.Context
-import android.content.pm.PackageManager
-import com.avito.logger.Logger
 import com.google.android.gms.common.ConnectionResult
 import com.google.android.gms.common.GoogleApiAvailability
 
-internal fun Context.checkPlayServices(logger: Logger) {
-    logger.debug(
-        "Required play services version: $playServicesMetaDataVersion, on device: $playServicesOnDeviceVersion"
```

Comment:
1. Don't need this log when everything is ok.
In case of error, it's also useless. We can't choose the version, only update for the latest.

2. Logger is broken in infra tests here due to MBS-11782.